docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -32,6 +32,12 @@
             <td><p>Enum</p></td>
             <td>Whether to double write to a changelog file. This changelog file keeps the details of data changes, it can be read directly during stream reads.<br /><br />Possible values:<ul><li>"none": No changelog file.</li><li>"input": Double write to a changelog file when flushing memory table, the changelog is from input.</li><li>"full-compaction": Generate changelog files with each full compaction.</li></ul></td>
         </tr>
+        <tr>
+            <td><h5>changelog-producer.full-compaction.trigger-interval</h5></td>
```

Comment:
I think we can make this key shorter.
Maybe just `changelog-producer.compaction-interval`?